lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,32 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # class to feel yourself real driver
+  class Car
+    attr_reader :color, :model, :current_speed, :year
+
+    def self.default_car
+      new({})
+    end
+
+    def initialize(args)
```

Comment:
in this case i will have an error
Car::#initialize#test_0001_expects hash of attributes